README.md

```diff
@@ -150,8 +150,8 @@ Ensure you have installed correct versions of necessary dependencies:
 * For `tsh` version > `10.x` with FIDO2 support, you will need `libfido2` and
   `pkg-config` installed locally
 * To build the web UI:
-  * [`yarn`](https://classic.yarnpkg.com/en/docs/install)(< 2.0.0) is required.
-  * If you prefer not to install/use yarn, but have docker available, you can run `make docker-ui` instead.
+  * [`pnpm`](if you have Node.js installed, run `corepack enable` to make `pnpm` available).
```

Comment:
> To use pnpm, enable Corepack (if you haven't already done it) with `corepack enable`.
> All commands stayed the same, you only need to change `yarn` to `pnpm`, e.g. `pnpm start-teleport`.
> The only important difference is dependencies installation, previously you could type `yarn` now it's `pnpm install` (or `pnpm i`).

Is it going to break workflows of backend devs who happen to run `make build/teleport`?

Reply:
Yes, `make build/teleport` calls `yarn install`, so after switching to `pnpm` it will invoke `pnpm install`. If pnpm is not installed, it will fail. 
I will post instructions on slack about enabling corepack.

Reply:
Could we then somehow make it so that the workflow won't be broken? Like by calling `corepack enable pnpm` automatically if corepack is installed? idk what would actually help here and if majority of them have corepack installed. But if we can avoid breaking the dev env for 5+ people, I think we should aim to avoid doing that.

Reply:
Yeah, I think autoenabling pnpm would help, I added it in  https://github.com/gravitational/teleport/pull/44167/commits/fe09f7175ae49352f927d742e30ca1bd674b6cbc. Please let me know what you think.